package.json

```diff
@@ -176,6 +177,7 @@
     "rxjs": "^6.4.0",
     "source-map-support": "^0.5.11",
     "uniqid": "^5.0.3",
+    "uuid": "^3.3.2",
```

Comment:
Already have a dependency for generating unique ids (uniqid). Is uuid necessary?